src/ceph-disk/ceph_disk/main.py

```diff
@@ -1498,7 +1532,23 @@ def create_partition(self, uuid, name, size=0, num=0):
         if num == 0:
             num = get_free_partition_index(dev=self.path)
         if size > 0:
-            new = '--new={num}:0:+{size}M'.format(num=num, size=size)
```

Comment:
of course, should I make it into a separate commit but in this PR or into a new PR?